tests/stress/stress_test.go

```diff
@@ -111,13 +148,6 @@ func Benchmark20Environments20Deploys(b *testing.B) {
 	})
 }
 
-func Benchmark5Environments50Deploys(b *testing.B) {
-	runTest(b, StressTestCase{
-		NumDeploys:      50,
-		NumEnvironments: 5,
-	})
-}
-
 func Benchmark5Environments100Deploys(b *testing.B) {
```

Comment:
`Benchmark5Environments100Deploys` is an odd metric to test. Deploys are totally independent of environments, why wouldn't just test deploys by themselves? 

I think the **only** deploy-related benchmark tests should be:
`BenchmarkNDeploysAcrossXFamilies`